ixmp/core/platform.py

```diff
@@ -162,8 +155,8 @@ def get_log_level(self) -> str:
     def scenario_list(
         self,
         default: bool = True,
-        model: Optional[str] = None,
-        scen: Optional[str] = None,
+        model: str | None = None,
+        scen: str | None = None,
```

Comment:
```suggestion
        scen: str | None = None
```
I'm wondering if these annotations are now short enough to fit into a single line. If you remove the comma as in the suggestion, does ruff change the formatting or just re-introduce the comma?